dev-tools/phpstan/baseline.php

```diff
@@ -523,6 +523,12 @@
 	'count' => 2,
 	'path' => __DIR__ . '/../../src/DocBlock/TypeExpression.php',
 ];
+$ignoreErrors[] = [
```

Comment:
this added baseline entry is fine and should be fixed with/after https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8103